examples/pylab_examples/scatter_profile.py

```diff
@@ -9,16 +6,16 @@
 10000    3.30           1.31            0.53
 50000    19.30          6.53            1.98
 """
-from __future__ import print_function
-import pylab
+from __future__ import print_function  # only needed for python 2.x
+import matplotlib.pyplot as plt
+import numpy as np
 
 import time
 
-
 for N in (20, 100, 1000, 10000, 50000):
     tstart = time.time()
-    x = 0.9*pylab.rand(N)
-    y = 0.9*pylab.rand(N)
-    s = 20*pylab.rand(N)
-    pylab.scatter(x, y, s)
+    x = 0.9*np.random.random(N)
```

Comment:
Ack! I will get that changed as soon as I get back to my computer!



Cheers,Eric

On Sun, Jul 19, 2015 at 1:53 PM, Jens Hedegaard Nielsen
<notifications@github.com> wrote:

>>  for N in (20, 100, 1000, 10000, 50000):
>>      tstart = time.time()
>> -    x = 0.9*pylab.rand(N)
>> -    y = 0.9*pylab.rand(N)
>> -    s = 20*pylab.rand(N)
>> -    pylab.scatter(x, y, s)
>> +    x = 0.9*np.random.random(N)
> Sorry it has to be np.random.rand
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/matplotlib/matplotlib/pull/4734/files#r34960322